### PR TITLE
CLDR-15369 Priority Items Summary snapshots, improvements

### PR DESCRIPTION
**Pull request by btangmu**
-For snapshots, include only locales in SubmissionLocales.CLDR_LOCALES

-Fix snapshot schedule in case server is started between midnight and 1:00 AM

-Remove the word slow from the menu item

-Automatically show the most recent summary snapshot

-Display controls below the summary rather than above it

-Unrelated formatting change for Test.html (forgot to run prettier for favicon PR)

CLDR-15369

- [ ] This PR completes the ticket.

### CODE REVIEW

**Comment by btangmu** [comment]
@macchiati @srl295 if there's no objection, after this is merged, I'll set CLDR_AUTO_SNAP=true again in cldr.properties on cldr-smoke. Then we should start getting auto snapshots at one o'clock every morning, and if those work OK we can do the same on the production server -- that might complete the ticket

**Comment by macchiati** [comment]
Sgtm

File: tools/cldr-apps/js/src/esm/cldrText.js [hunk 1]
```diff
@@ -467,7 +467,7 @@ const strings = {
   special_users: "Users",
   special_vetting_participation: "Vetting Participation",
   special_vetting_participation2: "Vetting Participation v2",
-  special_vsummary: "Priority Items Summary (slow)",
```
Comment (srl295) [inline tools/cldr-apps/js/src/esm/cldrText.js:470]: Maybe when CLDR-14543 lands we can change this to `(fast)` :)

**Review by srl295 — APPROVED** [review]

File: tools/cldr-apps/js/src/esm/cldrText.js (hunk 1 above)
Comment (btangmu) [reply tools/cldr-apps/js/src/esm/cldrText.js:470]: good idea

**Pull request merged**